### PR TITLE
Add the-bitcoin-development-kit-bdk review to chaincode-labs/chaincode-podcast

### PR DESCRIPTION
**Pull request by nymius**
This PR adds [the-bitcoin-development-kit-bdk](https://podcasters.spotify.com/pod/show/chaincode/episodes/Alekos-Filini--Daniela-Brozzoni-and-the-Bitcoin-Development-Kit-BDK---Episode-32-e24m4o1) transcript review to the chaincode-labs/chaincode-podcast directory.

### CODE REVIEW

File: chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md [hunk 1]
```diff
@@ -328,39 +319,45 @@ So we kind of explored the ideas for...
 So at one point I wanted to make BDK generic over the amount type.
 So the idea was to use Rust generic so that you can have...
 You can use any type for your amount as long as you can basically sum two amounts so you can compare them so I can say this is greater than that.
-So the idea was if I, for Bitcoin you would use an unsigned 64-bit integer but for liquid you would use their, whatever, Pedersen commitment type, because you can sum them together, but you can't really compare them, so you can't say is this greater than this one because it's binary, so I tried to implement that and it was a mess, so I stopped.
+So the idea was if I, for Bitcoin you would use an unsigned 64 bit integer but for liquid you would use their, whatever, Pedersen commitment type, because you can sum them together, but you can't really compare them, so you can't say is this greater than this one because it's binary, so I tried to implement that and it was a mess, so I stopped.
 So yeah, I think that that's one of the roadblocks, otherwise they would probably be happy to use it, I think.
+
+Adam Jonas & Mark Erhardt: 00:16:21
```
Comment (kouloumos) [inline chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md]: I appreciate this, but we don't really support this kind of "double" speaker attribution. I removed one speaker to stick with the supported format.

File: chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md [hunk 2]
```diff
@@ -328,39 +319,45 @@ So we kind of explored the ideas for...
...
+
 Interesting.
+
+Alekos Filini: 00:16:22
+
 Yeah.
```
Comment (kouloumos) [inline chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md]: Fill free to delete this kind of non essential lines.

File: chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md [hunk 3]
```diff
@@ -180,68 +171,68 @@ So in terms of users, we don't have that many.
...
+Essentially we wanted to make something super generic so we focused both on in terms of features like we want to provide an API that can be, I don't know if you want to build a single-sig wallet you can use it easily, But if you want to build a multi-sig wallet, you can do the same, as easy as building a single-sig wallet.
 Or even if you want to start doing more advanced stuff, you want to have time locks in your scripts.
 The idea we wanted to build was something that can kind of generalize.
 And then you just use one API, and everything works fine.
 And this is in terms of features and in terms of like platforms, the library works well on desktop, works on mobile.
 Now with the refactoring of BDK 1.0 it's going to work on embedded hardware as well.
-So it's going to be no STD in technical terms.
-So, yeah, the idea is basically to build something that works anywhere and can kind of do anywhere, everything.
+So it's going to be no std in technical terms.
```
Comment (kouloumos) [inline chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md]: Feel free to use backticks for this kind of code related terms.
```suggestion
So it's going to be no `std` in technical terms.
```

File: chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md [hunk 4]
```diff
@@ -259,63 +250,63 @@ Although I would argue that it's a good investment to spend time and learn BDK b
 You want to introduce remote signers with 2FA or stuff like that and with BDK it's super easy to do.
 So I don't know, if I were developing something I would do that extra step, maybe some other developers are not.
 
-Speaker 2: 00:12:17
+Daniela Brozzoni: 00:12:17
 
 Yeah, I think we have some work to do on the documentation side of things, because I realized recently just looking at the questions that people had, that our documentation assumed that people would have a good knowledge of how Bitcoin works.
-And if you're building a wallet, I mean, you should know something about Bitcoin, of course, but you don't have to know exactly what's a C-cash and what's a lock time, etc.
+And if you're building a wallet, I mean, you should know something about Bitcoin, of course, but you don't have to know exactly what's a ccache and what's a lock time, etc.
```
Comment (kouloumos) [inline chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md]: Feel free to use backticks for this kind of code related terms.
```suggestion
And if you're building a wallet, I mean, you should know something about Bitcoin, of course, but you don't have to know exactly what's a `ccache` and what's a lock time, etc.
```

**Review by kouloumos — COMMENTED** [review]
Thank you for your contribution @nymius! You did amazing work on catching AI's transcription mistakes. 

I left a couple of comments inline on places were you could add backticks for clarity on code-related technical terms. I fixed those on [my fixup commit](https://github.com/bitcointranscripts/bitcointranscripts/pull/474/commits/0d8ed58602a20776700b408bf1a69cca4115a350) so there is no extra work for you to do as part of this submission. I also added a couple of examples on improvement that you could do on transcription style - removing false starts, filler words, etc.

All in all, great first submission! :rocket: As you continue with your review work, feel free to join our [Bitcoin Dev Project Discord](https://discord.gg/76wCxgeyvM) to ask questions, connect with other reviewers, and stay updated on new transcripts available for review. This community can be a great resource as you grow as a contributor.

Hopefully we will see more transcripts from you in the future!

**Comment by nymius** [comment]
Thanks for the review @kouloumos, will consider your comments for future transcripts.

File: chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md [hunk 5]
```diff
@@ -112,15 +114,15 @@ And he really did like a lot of contribution.
 He was using it for, I don't remember his name, GAN, I think, something like that.
```
Comment (nymius) [inline chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md:114]: I couldn't avoid to think this `GAN` transcription was wrong. Luckily, I stumbled on the actual project a couple of days ago: https://github.com/GoUpNumber/gun. Probably the inclusion of hyperlinks goes beyond the objectives of transcription, but at least we can change it to \`gun\` , WDYT @kouloumos ?

Comment (kouloumos) [reply chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md:114]: Of course, anybody can open a PR to make a change (small or big) to a transcript.

**Pull request merged**